tsconfig.json

```diff
@@ -1,15 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+    "allowJs": true,
+    "target": "es3",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
     "declaration": true,                   /* Generates corresponding '.d.ts' file. */
     "outDir": "./dist",                        /* Redirect output structure to the directory. */
```

Comment:
as I understand you removed `dist` in this pr.

Reply:
Yep.